### PR TITLE
[DEV-2008] Add the new 'is_duplicate' param description and general fixes

### PR DESCRIPTION
**Pull request by muhsen84**
# Contents of the *Pull Request*
- Add the new 'is_duplicate' param description
- Fix the copyright year to be the current year instead of a hardcoded value
- Fix the Delete Job page to show that jobs can be deleted at any stage
- Fix the Modify Job API endpoin

## Jira Tasks Associated

This PR is associated with the following Jira Tasks:

[DEV-2008] [API] Add parameter to the job creation API endpoint to bypass the duplicate detection

[DEV-2008]: https://cielo24.atlassian.net/browse/DEV-2008

### CODE REVIEW

File: api/source/job_control/add_media_to_job.rst [hunk 1]
```diff
@@ -58,6 +58,18 @@ specify the Content-Length in the header. File size is limited to 10 gb.
 |                  | `Example`        | ``media_url=http%3A%2F%2Fwww.domain.com%2Fvideo.mp4``     |
 +------------------+------------------+-----------------------------------------------------------+
 
+**Query String Parameters** — Optional `(when adding media from URL)`
+
++------------------+------------------------------------------------------------------------------+
+| Name             | Details                                                                      |
++==================+==================+===========================================================+
+| is_duplicate     | `Description`    | Bypass the ducplicate detection settings                  |
```
Comment (jacksonpradolima) [inline api/source/job_control/add_media_to_job.rst:66]:
```suggestion
| is_duplicate     | `Description`    | Allow creating a job with the same settings that another job (duplicated job) |
```

File: api/source/job_control/add_media_to_job.rst [hunk 2]
```diff
@@ -58,6 +58,18 @@ specify the Content-Length in the header. File size is limited to 10 gb.
...
+|                  +------------------+-----------------------------------------------------------+
+|                  | `Allowed Values` | Boolean                                                   |
```
Comment (jacksonpradolima) [inline api/source/job_control/add_media_to_job.rst:68]: Is boolean case insensitive? Maybe you can make this explicit

File: api/source/job_control/create_job.rst [hunk 3]
```diff
@@ -78,6 +78,12 @@ Creating a job is prerequisite for virtually all other methods.
 |                   +------------------+-----------------------------------------------------------+
 |                   | `Example`        | ``expected_speakers=40``                                  |
 +-------------------+------------------+-----------------------------------------------------------+
+| is_duplicate      | `Description`    | Bypass the ducplicate detection settings                  |
```
Comment (jacksonpradolima) [inline api/source/job_control/create_job.rst:81]:
```suggestion
| is_duplicate      | `Description`    | Bypass the duplicate detection settings                  |
```

Comment (jacksonpradolima) [reply api/source/job_control/create_job.rst:81]: Here you can use the same phrase that the suggested one: `Allow creating a job with the same settings that another job (duplicated job`

**Review by jacksonpradolima — CHANGES_REQUESTED** [review]

File: api/source/job_control/add_media_to_job.rst (hunk 1 above)
Comment (jacksonpradolima) [reply api/source/job_control/add_media_to_job.rst:66]: ducplicate  -> duplicate

Comment (muhsen84) [reply api/source/job_control/add_media_to_job.rst]: Addressed

File: api/source/job_control/add_media_to_job.rst (hunk 2 above)
Comment (muhsen84) [reply api/source/job_control/add_media_to_job.rst]: It is not case-sensitive, addressed

File: api/source/job_control/create_job.rst (hunk 3 above)
Comment (muhsen84) [reply api/source/job_control/create_job.rst]: Addressed

**Review by jacksonpradolima — APPROVED** [review]

**Pull request merged**